### ISSUE TITLE
Not support `chcp 936` 

### ISSUE THREAD

**Issue by aliuq** [microsoft/inshellisense#42]
**Describe the bug**

if the `$profile` contains `chcp 936`, it will be get a error from terminal

```bash
node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: ENOENT: no such file or directory, mkdir 'D:\xxxxxxx\src\�����ҳ: 936
C:\Users\xxxxxx\Documents\PowerShell'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'mkdir',
  path: 'D:\\xxxxxxxxxxxxx\\src\\�����ҳ: 936\r\n' +
    'C:\\Users\\xxxxxxxxxxxxx\\Documents\\PowerShell'
}

Node.js v18.16.1
```

**Expected behavior**
No error

**Environment**
 - OS: `Windows 11`
 - Output of `inshellisense --version`: `0.0.1-rc.3`
 - Nodejs Version: `v18.16.1`

**Additional context**

**Comment by cpendery** [comment]
@aliuq can you give me your anonymized `$profile` so I can replicate this?

**Comment by aliuq** [comment]
@cpendery Hi,

1. I commented out everything in `$profile`
2. Keep only `chcp 935`
3. Execute `pwsh` at the command line
4. Execute `is bind`, select pwsh, and throw an error message

```ps1
# C:\Users\xxxxxx\Documents\PowerShell\Microsoft.PowerShell_profile.ps1
chcp 935
```

**Comment by cpendery** [comment]
Closing as this was fixed in [0.0.1-rc.5](https://github.com/microsoft/inshellisense/releases/tag/0.0.1-rc.5)
